packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -192,6 +194,12 @@ export class CallContext {
         draft.calls[latestCallId] = call;
       }
     });
+    /**
+     * We want to remove the incoming call that matches the call if there is one
```

Comment:
nit: use imperative tense for comments, e.g.
```suggestion
      * Remove the incoming call that matches the call if there is one
```